CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 # Copyright (c) 2015 Thomas Heller
 # Copyright (c) 2015 Dominic Marcello
+# Copyright (c) 2018 Parsa Amini
 #
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-project(octotiger C CXX Fortran)
+project(octotiger CXX)
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
```

Comment:
Update to HPX minimal? Maybe we have to update ciricle-ci too?

Reply:
Done.